src/components/elements/RadioButtonGroup.jsx

```diff
@@ -12,13 +18,17 @@ function RadioButtonGroup ({ options, name, value }) {
               checked={value === option.value}
               type="radio"
               id={id}
+              data-testid={`radio-button-${name}-${option.value}`}
               name={name}
               value={option.value}
             />
             <label htmlFor={id}>
-              { option.label }
+              {`${ option.label }`}
```

Comment:
I think this extra interpolated is redundant unless we plan on 
adding something like this `label: ${ option.label }`